designs/embed_eksd_manifest.md

```diff
@@ -0,0 +1,59 @@
+# Embed EKS-D resources in EKS-A cluster
+
+## Introduction
+
+**Problem:** Currently, the EKS Anywhere controller pulls the EKS Distro manifest from the release URL provided in the versions bundle, which requires access to the internet.
+It would be ideal to have this manifest and its CRDs embedded in the cluster to allow the controller to operate without the requirement to make that external call.
+This will eventually be needed for air-gapped support.
```

Comment:
Can't we just dump all this stuff in the private registry?

Reply:
We could but I think this is more of a smaller step to fix this specific problem of downloading on the fly from within the controller, which was something that we wanted to fix from a previous issue